cypress/integration/workflow-executions.spec.js

```diff
@@ -106,7 +106,7 @@ describe('Workflow Executions List', () => {
         );
 
         cy.get(
-          `[href="/namespaces/default/workflows/${workflowId}/${runId}"] > :nth-child(2) > .table-link`,
+          `[href="/namespaces/default/workflows/${workflowId}/${runId}"] > :nth-child(3) > .table-link`,
```

Comment:
Maybe we add a better selector on to the table rows? We are already probably using some kind of index in the `{#each}` block in Svelte.